### PR TITLE
Getting started in new docs hub

### PR DESCRIPTION
**Pull request by pipo02mix**
### What this PR does / why we need it

Towards https://github.com/giantswarm/giantswarm/issues/30831

### Things to check/remember before submitting

- If you made content changes

    - Run `make dev` to render and proofread content changes locally.
    - Bump `last_review_date` in the front matter header if you reviewed the entire page.

### CODE REVIEW

File: src/content/getting-started/prepare-your-provider-account/azure/_index.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ If your current quotas are insufficient, you can request an increase. In the Azu
 
 You can follow the request status in the **Quotas** section.
 
-### Permissions
+## Step 2:## Permissions
```
Comment (pipo02mix) [inline src/content/getting-started/prepare-your-provider-account/azure/_index.md:49]:
```suggestion
## Step 2: Permissions
```

File: src/content/getting-started/prepare-your-provider-account/azure/_index.md [hunk 2]
```diff
@@ -156,7 +158,7 @@ az vm image terms accept --offer flatcar-container-linux-free --plan stable --pu
 
 **Note**: This acceptance needs to be performed only once for a subscription that's used to run Giant Swarm workload clusters.
 
-#### Enable encryption at host
+### Enable encryption at host
 
 If your security requirements demand this, there is a possibility to enable [encryption of data stored a virtual machine hosts](https://learn.microsoft.com/en-us/azure/virtual-machines/linux/disks-enable-host-based-encryption-cli). With this enabled, your data will be stored and transferred between storage and host encrypted, adding additional protecting from unauthorized access to your data.
```
Comment (lyind) [inline src/content/getting-started/prepare-your-provider-account/azure/_index.md:163]:
```suggestion
If your security requirements demand this, there is a possibility to enable [encryption of data stored on virtual machine hosts](https://learn.microsoft.com/en-us/azure/virtual-machines/linux/disks-enable-host-based-encryption-cli). With this enabled, your data will be stored and transferred between storage and host encrypted, adding additional protection from unauthorized access.
```

File: src/content/getting-started/prepare-your-provider-account/azure/_index.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ In addition to the following prerequisites, your account engineer will provide y
     * An account with [Owner](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#owner) or [User Access Administrator](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#user-access-administrator) role.
     * [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) installed and configured to point to chosen subscription via [`az account set`](https://learn.microsoft.com/en-us/cli/azure/account?view=azure-cli-latest#az-account-set) command.
 
-### Service quotas {#quotas}
+## Step 1: Service quotas {#quotas}
 
 Azure enforces [service quota through all cloud services](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits). The following overview lists the usual resources you may adjust depending on number of clusters and their size planned to be deployed:
```
Comment (lyind) [inline src/content/getting-started/prepare-your-provider-account/azure/_index.md:29]:
```suggestion
Azure enforces [service quota through all cloud services](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits). The following overview lists the usual resources you may adjust, depending on number and size of clusters planned to be deployed:
```

**Review by lyind — APPROVED** [review]
Minor changes, 𝕃𝔾𝕋𝕄

File: src/content/getting-started/prepare-your-provider-account/azure/_index.md [hunk 4]
```diff
@@ -125,26 +125,28 @@ az role assignment create \
     * Go to "Access control (IAM)"
     * Click "Add Role Assignment" and add the "Contributor" role and the "User Access Administrator" role to the App with the "subscription" Scope
 
-#### Step 2 - Provide generated credentials to Giant Swarm
+#### Provide generated credentials to Giant Swarm
 
 The following information needs to be provided to Giant Swarm:
 
+```
```
Comment (pipo02mix) [inline src/content/getting-started/prepare-your-provider-account/azure/_index.md:132]:
```suggestion
```text
```

File: src/content/overview/fleet-management/cluster-management/cluster-concepts/cluster-naming/_index.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+---
+title: Cluster naming
+description: The specifications for naming clusters in the Giant Swarm platform.
```
Comment (pipo02mix) [inline src/content/overview/fleet-management/cluster-management/cluster-concepts/cluster-naming/_index.md:3]:
```suggestion
description: The specifications for naming workload clusters in the Giant Swarm platform.
```

**Pull request merged**